hive-tables-creation.py

```diff
@@ -1,36 +1,52 @@
 import boto3
+import csv
+
+collections = []
+DatabaseName = "analytical_dataset_generation"
+with open("collections.csv", "rb") as csvfile:
+    reader = csv.reader(csvfile, delimiter=",")
+    for row in reader:
+        collections.append(row[0].replace(":", "_"))
 
 client = boto3.client("glue")
-client.delete_table(
-    DatabaseName="analytical_dataset_generation", Name="core_contract_hbase"
-)
-client.create_table(
-    DatabaseName="analytical_dataset_generation",
-    TableInput={
-        "Name": "core_contract_hbase",
-        "Description": "Hive table to access hbase table core:contract",
-        "StorageDescriptor": {
-            "Columns": [
-                {"Name": "rowkey", "Type": "string"},
-                {"Name": "data", "Type": "string"},
-            ],
-            "Location": "s3://${bucket}/analytical-dataset/hive/external/core_contract_hbase",
-            "Compressed": False,
-            "NumberOfBuckets": -1,
-            "SerdeInfo": {
-                "Name": "string",
-                "SerializationLibrary": "org.apache.hadoop.hive.hbase.HBaseSerDe",
-                "Parameters": {
-                    "hbase.columns.mapping": ":key,cf:record",
-                    "serialization.format": "1",
+
+
+for collection in collections:
```

Comment:
Looks like the code can be optimised  with in 1 for loop

Reply:
done